eox_audit_model/models.py

```diff
@@ -98,7 +98,7 @@ class AuditModel(models.Model):
     method_name = models.CharField(max_length=150)
     output_parameters = models.TextField(null=True, blank=True)
     performer = models.CharField(max_length=150, default=get_current_performer)
-    site = models.ForeignKey(Site, default=get_current_site)
+    site = models.ForeignKey(Site, default=get_current_site, on_delete=models.CASCADE)
```

Comment:
This is fixed now. Thank you

Reply:
I changed it to SET_NULL. Do you think this is ok? I could also create a method that returns the 'Missing domain.' site as the default value in case the referenced site gets deleted.  @andrey-canon